authors/apps/profiles/serializers.py

```diff
@@ -3,6 +3,7 @@
 from .models import Profile
 from authors.apps.authentication.models import User
 
+
```

Comment:
@armstrongsouljah unnecessary change here

Reply:
This happened due to a rebase that led to the loss of my code. Thus the change.